ArchiSteamFarm/Bot.cs

```diff
@@ -2521,6 +2564,16 @@ internal sealed class Bot : IDisposable {
 							Utilities.InBackground(Trading.OnNewTrade);
 						}
 
+						break;
+					case ArchiHandler.UserNotificationsCallback.EUserNotification.Gifts:
+						bool newGifts = notification.Value > GiftsCount;
+						GiftsCount = notification.Value;
+
+						if (newGifts) {
```

Comment:
One last request: add `&& BotConfig.AcceptGifts` to this `if` so ASF doesn't redeem gift cards automatically. I'll update wiki to cover this.